dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/test/java/org/apache/dolphinler/plugin/task/mlflow/MlflowTaskTest.java

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.dolphinler.plugin.task.mlflow;
 
+import static org.powermock.api.mockito.PowerMockito.when;
```

Comment:
Hello @jieguangzhou , could u plz remove `powermock` in this test class? For reason why we need to do this, see: #11405 . You could refer to  #11588 as an example.

Reply:
That's good advice, I wil modify some ML task plugin test cases in a new PR

Reply:
@jieguangzhou do not forget create some issue to target remove power mock to all mlops tasks

Reply:
I have already created a new issue about that #12077

Reply:
@jieguangzhou FYI, we just added jUnit5 dependency #11332 this morning. Therefore, I will update the example I posted above with jUnit5 later this week.

Reply:
Thank you. I'll write with reference